sql-plugin/src/main/scala/com/nvidia/spark/rapids/RapidsConf.scala

```diff
@@ -1189,10 +1236,41 @@ class RapidsConf(conf: Map[String, String]) extends Logging {
 
   lazy val getCloudSchemes: Option[Seq[String]] = get(CLOUD_SCHEMES)
 
+  lazy val optimizerEnabled: Boolean = get(OPTIMIZER_ENABLED)
+
+  lazy val optimizerExplain: String = get(OPTIMIZER_EXPLAIN)
+
+  lazy val optimizerDebugEnabled: Boolean = get(OPTIMIZER_DEBUG_ENABLED)
```

Comment:
This config appears to be unused?  Also I would expect any debug logging to be done with the `logDebug` method from the `Logging` interface, otherwise we're going to get a plethora of debug logging enable configs for stuff over time.

Reply:
Yes, I was using this at one point. I've removed it now.